<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAINPU1502-0014" name="Suacelab_File_Upload_Demo" tests="1" failures="0" timestamp="23 Dec 2020 10:15:28 GMT" time="40.964" errors="1">
  <testcase name="FileUpload_UsingSikuli" time="40.964" classname="Suacelab_File_Upload_Demo">
    <error type="org.sikuli.script.FindFailed" message="C:\OrangeHRM_Gradle\Sikuli_Objects\FileTextBox.PNG: (741x38) in R[0,0 1366x768]@S(0)">
      <![CDATA[FindFailed: C:\OrangeHRM_Gradle\Sikuli_Objects\FileTextBox.PNG: (741x38) in R[0,0 1366x768]@S(0)
  Line 2222, in file Region.java

	at org.sikuli.script.Region.wait(Region.java:2222)
	at Suacelab_File_Upload_Demo.FileUpload_UsingSikuli(Suacelab_File_Upload_Demo.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- FileUpload_UsingSikuli -->
</testsuite> <!-- Suacelab_File_Upload_Demo -->
